staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go

```diff
@@ -143,6 +144,9 @@ func (intstr *IntOrString) Fuzz(c fuzz.Continue) {
 }
 
```

Comment:
If you had a 

```go
func NewZero() *IntOrPercent{
    return the equivalent of zero
}

ValOrDefault(intOrPercent *IntOrString, defaultValue *IntOrString){
  if intOrPercent == nil{
       return defaultValue
    }
}
```

Then you could use it in the above case and whereever it is in downstream too, right?

Reply:
Yeah, that make sense. Althogh `NewZero` is not needed we have `FromInt` and `FromString` helpers that should do it.